### PR TITLE
Feature/175

### PR DESCRIPTION
**Pull request by OksanaSam**
#175 

- Removed the save button;
-  Added the logic associated with the save button to the next button;
- Fixed tests to account for changes

### CODE REVIEW

File: web-client/src/shared/components/drivers-registration/CurrentSlide/index.tsx [hunk 1]
```diff
@@ -111,22 +91,26 @@ const DriversRegistrationCurrentSlide = (): React.ReactElement => {
...
+                        {activeStep === steps.length - 1 ? (
+                            <Box mr={2}>
+                                <Button variant="contained" onClick={handleBack}>
+                                    Back
+                                </Button>
+                            </Box>
+                        ) : 
+                           null
+                        }
                         {activeStep === steps.length - 1 ? (
```
Comment (ynynl) [inline web-client/src/shared/components/drivers-registration/CurrentSlide/index.tsx:106]: Is it repeating the above line?

File: web-client/src/shared/components/drivers-registration/PreScreenRequirements/index.tsx [hunk 2]
```diff
@@ -57,8 +58,15 @@ const PreScreenRequirements = ({ onSubmit }: PreScreenPorps): React.ReactElement
   const policeCheck = watch('policeRecordsCheckOptions')
   const drivingAbstract = watch('drivingAbstractOptions')
 
+
+  const handlePreScreenSubmit = (data: PreScreen) => {
+    const newState = { ...preScreen, ...data }
+    dispatch(setPreScreen(newState))
+    handleNext();
```
Comment (ynynl) [inline web-client/src/shared/components/drivers-registration/PreScreenRequirements/index.tsx:65]: Perfect!!!!

**Review by ynynl — APPROVED** [review]
Everything looks great!

**Comment by OksanaSam** [comment]
All lint checks and tests run locally without errors

**Comment by l2silver** [comment]
@OksanaSam looks like this one has some cypress tests are failing. Any chance you'll have time to take a look at these this week?

**Comment by OksanaSam** [comment]
> @OksanaSam looks like this one has some cypress tests are failing. Any chance you'll have time to take a look at these this week?

Ok, I will look at them

**Comment by OksanaSam** [comment]
Hey @l2silver It should be good now. All checks pass except for PR Github, I don't really know how to fix that.

**Comment by l2silver** [comment]
> Hey @l2silver It should be good now. All checks pass except for PR Github, I don't really know how to fix that.

No one does ;)

**Review by l2silver — APPROVED** [review]

**Pull request merged**